src/Microsoft.Net.Http.Headers/project.json

```diff
@@ -22,5 +22,8 @@
         "System.Runtime": "4.0.21-*"
       }
     }
+  },
+  "dependencies": {
+    "Microsoft.Extensions.Primitives": "1.0.0-*"
```

Comment:
This dependency is problematic given our long term plans for ownership of this package. /cc: @davidfowl